tools/src/wyvern/tools/parsing/coreparser/arch/ASTAttachmentDecl.java

```diff
@@ -0,0 +1,57 @@
+/* Generated By:JJTree: Do not edit this line. ASTAttachmentDecl.java Version 4.3 */
+/* JavaCCOptions:MULTI=true,NODE_USES_PARSER=false,VISITOR=true,TRACK_TOKENS=false,
+NODE_PREFIX=AST,NODE_EXTENDS=,NODE_FACTORY=,SUPPORT_CLASS_VISIBILITY_PUBLIC=true */
+package wyvern.tools.parsing.coreparser.arch;
+
+import java.util.HashSet;
+
+public class ASTAttachmentDecl extends SimpleNode {
+    private HashSet<String> ports = new HashSet<>();
```

Comment:
Please add a comment to describe what the elements of the ports set are. Include the fact that they are of the form component_instance.port.